Podfile

```diff
@@ -7,6 +7,8 @@ target 'Pokedex_ios' do
 
   # Pods for Pokedex_ios
   pod 'SwiftLint', '0.47.1'
+  
```

Comment:
Lthe podfile swp shouldn’t be added to the repo (is the file before but couldn’t comment on it)